CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Prowide Core - CHANGELOG
 
+#### 9.5.4 - SNAPSHOT
```

Markdownlint (0.35.0)

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)